app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
It is better to use the class name to get access to the class attributes.
you can read about it here [https://github.com/mate-academy/py-herbivores-and-carnivores/blob/master/checklist.md](url)